package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@clr/clarity",
-  "version": "1.0.4",
+  "version": "1.0.5",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
```

Comment:
I think I actually corrected these now!  (They were left over from a hacky publish to an internal npm proxy so that we could start using the feature).